### PR TITLE
REEEEEEEEEEEEEEEEEEEEEEEEEE

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
reeeeeeee

### CODE REVIEW

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -330,4 +330,4 @@
 	desc = "A kit with the necessary equipment to become an excellent martial artist, and terrible parent!"
 	item = /obj/item/storage/box/syndie_kit/armstrong
 	cost = 14
-	surplus = 20 // someone who respects the eldritch god Nar-Sie a little (((too much))) complained
+	surplus = 20 // someone who respects the eldritch god Nar-Sie a little (((too much))) complained
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/uplink/uplink_item.dm:333]: This changes absolutely nothing. Have no idea why git decided to add this as well.

File: hippiestation/code/game/objects/items/clowntools.dm [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/objects/items/clowntools.dm:1]: Why did git add this? Won't change anything.

File: hippiestation/code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/uplink/uplink_item.dm:333]: dirty commit log

Comment (Manly-Dwarf) [reply hippiestation/code/modules/uplink/uplink_item.dm:333]: Yeah well you're ugly

**Comment by Manly-Dwarf** [comment]
REEEEEEEEEEEEEEE

**Pull request closed without merging**